package.json

```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "start": "pm2 start process.json",
     "dev": "pm2 start process-dev.json",
+    "local": "nodemon ./dist/src/app.js",
+    "build": "tsc",
```

Comment:
have this run post install

Reply:
@tsingh777 you mean `tsc && npm run start` ? I don't have process(-dev).json file to test it.

Reply:
add  a `"postinstall": "npm run test",`

Reply:
Done